### PR TITLE
feat: add cache icons for cached tasks in graph view

### PR DESCRIPTION
**Pull request by olga-union**
Signed-off-by: Olga Nad <olga@union.ai>
## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Added icons to indicate cache status of the task in graph view.
Removed unused code `components/flytegraph/Graph.tsx`.

## Tracking Issue
fixes https://github.com/flyteorg/flyte/issues/300

## Follow-up issue
_NA_

## Demo
With read from cache tasks: Screen Shot 2022-04-18 at 12 12 21 PM
With written into cache tasks: Screen Shot 2022-04-18 at 12 11 35 PM

## Bonus
Fixed placement of sub headers in info section above the tabs.
Before: Screen Shot 2022-04-14 at 10 20 29 AM
After: Screen Shot 2022-04-18 at 12 12 48 PM

### CODE REVIEW

File: src/components/Executions/ExecutionDetails/ExecutionMetadata.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const useStyles = makeStyles((theme: Theme) => {
       position: 'relative',
     },
     detailsContainer: {
-      alignItems: 'center',
+      alignItems: 'start',
```
Comment (jsonporter) [inline src/components/Executions/ExecutionDetails/ExecutionMetadata.tsx:26]: nice 👍

**Review by jsonporter — APPROVED** [review]
lgtm

File: src/components/flytegraph/ReactFlow/customNodeComponents.tsx [hunk 2]
```diff
@@ -238,10 +247,24 @@ export const ReactFlowCustomTaskNode = ({ data }: any) => {
       </div>
     );
   };
+
+  const renderCacheIcon = (cacheStatus) => {
+    if (cacheStatus === CatalogCacheStatus.CACHE_HIT) {
+      return <CachedOutlined style={cacheIconStyles} />;
```
Comment (anrusina) [inline src/components/flytegraph/ReactFlow/customNodeComponents.tsx:253]: nit: switch could be a better choice from readability perspective

File: src/components/Executions/ExecutionDetails/ExecutionMetadata.tsx (hunk 1 above)
Comment (anrusina) [inline src/components/Executions/ExecutionDetails/ExecutionMetadata.tsx:26]: nit: Maybe it's only me, but I feel like icon should go 1-2 px lower, to align better with text (as it mainly uses lower case naming), plus it looks like in cases when cache icon present right margin after the icon should become smaller for 2px too.
Screen Shot 2022-04-18 at 12 32 29 PM

**Review by anrusina — DISMISSED** [review]
lgtm

File: src/components/flytegraph/ReactFlow/customNodeComponents.tsx (hunk 2 above)
Comment (olga-union) [reply src/components/flytegraph/ReactFlow/customNodeComponents.tsx:253]: rewrote it in switch this morning, but then decided to use ifs 😆

File: src/components/Executions/ExecutionDetails/ExecutionMetadata.tsx (hunk 1 above)
Comment (olga-union) [reply src/components/Executions/ExecutionDetails/ExecutionMetadata.tsx:26]: added `1px` margin top in `customNodeComponents.tsx`, where the icon styles are defined

File: src/components/flytegraph/ReactFlow/customNodeComponents.tsx (hunk 2 above)
Comment (olga-union) [reply src/components/flytegraph/ReactFlow/customNodeComponents.tsx]: updated!

**Review by anrusina — APPROVED** [review]
lgtm

**Pull request merged**